docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -195,22 +223,24 @@ Select a rule name from the rule listing to access the *Rule details* page, whic
 [role="screenshot"]
 image::images/rule-details-alerts-active.png[Rule details page with three alerts]
 
-In this example, the rule detects when a site serves more than a threshold number of bytes in a 24 hour period. Three sites are above the threshold. These are called alerts - occurrences of the condition being detected - and the alert name, status, time of detection, and duration of the condition are shown in this view.
+In this example, the rule detects when a site serves more than a threshold number of bytes in a 24 hour period. Four sites are above the threshold. These are called alerts - occurrences of the condition being detected - and the alert name, status, time of detection, and duration of the condition are shown in this view.
 
-Upon detection, each alert can trigger one or more actions. If the condition persists, the same actions will trigger either on the next scheduled rule check, or (if defined) after the re-notify period on the rule has passed. To prevent re-notification, you can suppress future actions by clicking on the switch to mute an individual alert.
-
-[role="screenshot"]
-image::images/rule-details-alert-muting.png[Muting an alert,width=50%]
+Upon detection, each alert can trigger one or more actions. If the condition persists, the same actions will trigger either on the next scheduled rule check, or (if defined) after the re-notify period on the rule has passed. To prevent re-notification, you can suppress future actions by clicking the toggle in the *Mute* column to mute an individual alert.
 
 Alerts will come and go from the list depending on whether they meet the rule conditions or not - unless they are muted. If a muted instance no longer meets the rule conditions, it will appear as inactive in the list. This prevents an alert from triggering actions if it reappears in the future.
 
+////
+TBD: Steps for creating "recovered alerts" 
 [role="screenshot"]
 image::images/rule-details-alerts-inactive.png[Rule details page with three inactive alerts]
 
+TBD: Is this now replaced by the "Snooze notifications" functionality?
 If you want to suppress actions on all current and future alerts, you can mute the entire rule. Rule checks continue to run and the alert list will update as alerts activate or deactivate, but no actions will be triggered.
 
+TBD: This toggle no longer seems to exist
 [role="screenshot"]
 image::images/rule-details-muting.png[Use the mute toggle to suppress all actions on current and future alerts,width=50%]
+////
```

Comment:
This section of the documentation seems to contain some functionality that is outdated (muting a rule).  It also contains a screenshot that I haven't been able to reproduce yet (recovered alerts).